installPhase.sh

```diff
@@ -15,11 +15,19 @@ else
     VERSION_APPENDIX="-NOGIT"
 fi
 
+# Extract version from Cargo.toml
+PACKAGE_VERSION=$(grep '^version =' Cargo.toml | sed -E 's/version = "(.*)" # x-release-please-version$/\1/')
```

Comment:
would using https://github.com/woodruffw/toml2json be cleaner here?

Reply:
I'm happy with both implementations. Maybe adding a new dependency just for a single line is overkill but thats just my opinion.